mapper/orm-coordination-outbox-polling/src/main/java/org/hibernate/search/mapper/orm/coordination/outboxpolling/cfg/HibernateOrmMapperOutboxPollingSettings.java

```diff
@@ -215,6 +215,19 @@ private HibernateOrmMapperOutboxPollingSettings() {
 	public static final String COORDINATION_EVENT_PROCESSOR_PULSE_EXPIRATION =
 			PREFIX + Radicals.COORDINATION_EVENT_PROCESSOR_PULSE_EXPIRATION;
 
+	/**
+	 * In the event processor, the order in which outbox events are processed.
+	 * <p>
+	 * Only available when {@value HibernateOrmMapperSettings#COORDINATION_STRATEGY} is
+	 * {@value #COORDINATION_STRATEGY_NAME}.
+	 * <p>
+	 * Expects one of the "external representation" strings defined in {@link OutboxEventProcessingOrder}.
+	 * <p>
+	 * Defaults to {@link Defaults#COORDINATION_EVENT_PROCESSOR_ORDER}.
+	 */
+	public static final String COORDINATION_EVENT_PROCESSOR_ORDER =
```

Comment:
just in case.. do we need a test for the new property?

Reply:
Yes, you're right, I forgot about that :/ Thanks for the heads-up.